apps/example/pages/legacy/[[...nextadmin]].tsx

```diff
@@ -0,0 +1,27 @@
+import { AdminComponentProps, NextAdmin } from "@premieroctet/next-admin";
```

Comment:
This is not legacy, it is still available and maintained in Next.js.
We should expose both with the same priority.

Reply:
What would you name it instead ?